core/src/main/java/io/opentelemetry/android/internal/session/SessionManagerImpl.kt

```diff
@@ -11,10 +11,14 @@ import io.opentelemetry.android.session.SessionIdGenerator
 import io.opentelemetry.android.session.SessionManager
 import io.opentelemetry.android.session.SessionObserver
 import io.opentelemetry.android.session.SessionStorage
+import io.opentelemetry.context.Context
+import io.opentelemetry.context.ContextKey
 import io.opentelemetry.sdk.common.Clock
 import java.util.Collections.synchronizedList
 import kotlin.time.Duration
 
+val SESSION_CONTEXT_KEY: ContextKey<Session> = ContextKey.named("rum.session")
```

Comment:
Why not just `session` like for the trace attribute?

Reply:
I'm open to it, but I wanted it to be explicit that it was the rum session and not any other session that might be in the context in the future....like a user session or a shopping session or I dunno.